Shared/UI/Transitions/ModalTransitionController.swift

```diff
@@ -21,11 +21,7 @@ class ModalTransitionController: NSObject, UIViewControllerTransitioningDelegate
 
         if let from = source as? TransitionableViewController,
             let toVC = presented as? TransitionableViewController {
-
-            // If there is a parent it will crash on dismiss.
-            if from.parent.isNil {
```

Comment:
Not sure why this was causing a crash. Doesnt in ours and parent is always the root.